core/domain/stats_services_test.py

```diff
@@ -1894,15 +1912,24 @@ class StateAnswersStatisticsTest(test_utils.GenericTestBase):
 
     def _get_top_state_answer_stats(
             self, exp_id=EXP_ID, state_name=STATE_NAMES[0]):
+        """Returns the top state answer stats corresponding to the given
+        exploration id and state names.
+        """
         return stats_services.get_top_state_answer_stats(exp_id, state_name)
 
     def _get_top_state_unresolved_answer_stats(
             self, exp_id=EXP_ID, state_name=STATE_NAMES[0]):
+        """Returns the top state unresolved answer stats corresponding to the
```

Comment:
```suggestion
        """Returns the top unresolved answer stats corresponding to the
```
I think the "state" part is covered by the "corresponding to ..." part of the docstring. So we can drop it from here I feel. Ditto, for all the above and below functions.

Reply:
Done